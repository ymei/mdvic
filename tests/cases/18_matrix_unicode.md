Inline matrix: $\begin{pmatrix} a & b \\ c & d \end{pmatrix}$.

Display:
$$\begin{pmatrix} 1 & 22 \\ 333 & 4 \end{pmatrix}$$

