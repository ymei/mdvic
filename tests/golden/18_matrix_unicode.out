Inline matrix: (  a   b ;  c   d  ).

Display: 
(  1     22  )
(  333   4   )

